src/control/zoom/ZoomControl.cpp

```diff
@@ -27,12 +27,18 @@ void ZoomControl::zoomOneStep(bool zoomIn, double x, double y)
 {
 	XOJ_CHECK_TYPE(ZoomControl);
 
+	if(this->zoomPresentationMode)
```

Comment:
This probably solves the ticket: Presentation mode not working?!

Reply:
Well, it's just disables Zoom during presentationMode.
Maybe we should do this rather in zoomSequenceChanges?

Reply:
I would do it at the earliest convenient position in the code, to decrease the overhead. But essentially it is negligible.